resources/sass/app.scss

```diff
@@ -1,5 +1,7 @@
 @import url('https://fonts.googleapis.com/css2?family=Lato&family=Source+Serif+Pro&display=swap');
 
+@import '~@wmde/wikit-vue-components/dist/wikit-vue-components.css';
```

Comment:
We probably also want to import the tokens css in here, so that we can use them in the components we create.

Reply:
I thought about it when I read the instructions but didn't know if I should. Good idea! Thanks for pointing it out.